source/install/self-managed-changelog.md

```diff
@@ -12,6 +12,9 @@ Latest Mattermost Releases:
 
 ## Release v6.4 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v6.4.2, released 2022-03-10**
+  - Mattermost v6.4.2 contains medium severity level security fixes. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) to this release is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.org/responsible-disclosure-policy/).
```

Comment:
```suggestion
  - Mattermost v6.4.2 contains medium severity level security fixes. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) to this release is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
```

Reply:
@amyblais - Would it be possible to update any internal templates we have that includes links to legacy domains, please? Redirects are in place; however, it would be ideal for us provide users with the latest URLs across our docs assets.

Reply:
Thanks, @amyblais!! Let me know if you need any of the latest URLs.